odoo/addons/base/models/res_currency.py

```diff
@@ -37,7 +37,7 @@ class Currency(models.Model):
     position = fields.Selection([('after', 'After Amount'), ('before', 'Before Amount')], default='after',
         string='Symbol Position', help="Determines where the currency symbol should be placed after or before the amount.")
     date = fields.Date(compute='_compute_date')
-    currency_unit_label = fields.Char(string="Currency Unit", help="Currency Unit Name")
+    currency_unit_label = fields.Char(string="Currency Unit", help="Currency Unit Name", translate=True)
     currency_subunit_label = fields.Char(string="Currency Subunit", help="Currency Subunit Name")
```

Comment:
Following the same logic then use `translate=True` for `Cents` too in the field `currency_subunit_label`